subprojects/common/report-api/src/main/kotlin/com/avito/report/ReportLinkGenerator.kt

```diff
@@ -0,0 +1,26 @@
+package com.avito.report
+
+import com.avito.report.model.TestName
+
+public interface ReportLinkGenerator {
+
+    public fun generateReportLink(
+        filterOnlyFailtures: Boolean = false,
+        team: String? = null
+    ): String
+
+    public fun generateTestLink(testName: TestName): String
+
+    public class NoOp(
```

Comment:
impl

Reply:
extracted to separate class with more meaningful name